test/README.md

```diff
@@ -54,69 +54,16 @@ Layout
 Running tests
 =============
 
-Executing Tests
----------------
-
-To run all tests (except "old" tests):
-
-    make test
-
-To run only _unit_ tests:
-
-    make unittest
-
-To run only _functional_ tests:
-
-    make functionaltest
-
-
-Legacy tests
-------------
-
-To run all legacy Vim tests:
-
-    make oldtest
-
-To run a *single* legacy test file you can use either:
-
-    make oldtest TEST_FILE=test_syntax.vim
-
-or:
-
-    make src/nvim/testdir/test_syntax.vim
-
-- Specify only the test file name, not the full path.
-
-
-Debugging tests
----------------
+To run all tests (except "old" tests)
 
-- You can set `$GDB` to [run tests under gdbserver](https://github.com/neovim/neovim/pull/1527).
-  And if `$VALGRIND` is set it will pass `--vgdb=yes` to valgrind instead of
-  starting gdbserver directly.
-- Hanging tests often happen due to unexpected `:h press-enter` prompts. The
-  default screen width is 50 columns. Commands that try to print lines longer
-  than 50 columns in the command-line, e.g. `:edit very...long...path`, will
-  trigger the prompt. In this case, a shorter path or `:silent edit` should be
-  used.
-- If you can't figure out what is going on, try to visualize the screen. Put
-  this at the beginning of your test:
-
-    ```lua
-    local Screen = require('test.functional.ui.screen')
-    local screen = Screen.new()
-    screen:attach()
-    ```
-
-  Afterwards, put `screen:snapshot_util()` at any position in your test. See the
-  comment at the top of `test/functional/ui/screen.lua` for more.
-
-Filtering Tests
----------------
+```console
```

Comment:
do not "update" all the shell code to be highlighted. It was in indented form because that's more readable as a text file. Highlighting is not very useful for these small snippets, anyway.